app/Http/api_routes.php

```diff
@@ -22,6 +22,11 @@
         $api->resource('page', 'App\Api\V1\Controllers\PageController');
         $api->post('page/{id}/approve', 'App\Api\V1\Controllers\PageController@approve');
         $api->post('page/{id}/reject', 'App\Api\V1\Controllers\PageController@reject');
+        //Incident types routes
+        $api->post('incidents/types/', 'App\Api\V1\Controllers\IncidentTypeController@store');
```

Comment:
take a look to the possibility to use `$api->resource('incidents/types')` instead of defining every route separated.

Reply:
I have changed to $api->resource restricting the methods that need the admin role.